docs/ml-features.md

```diff
@@ -1252,6 +1252,21 @@ DataFrame transformedData = transformer.transform(dataFrame);
 
 {% endhighlight %}
 </div>
+
+<div data-lang="python" markdown="1">
+{% highlight python %}
+from pyspark.ml.feature import ElementwiseProduct
+from pyspark.mllib.linalg import Vectors
+
+data = [(Vectors.dense([1.0, 2.0, 3.0]),), (Vectors.dense([4.0, 5.0, 6.0]),)]
+df = sqlContext.createDataFrame(data, ["vector"])
+transformer = ElementwiseProduct(scalingVec=Vectors.dense([0.0, 1.0, 2.0]), 
+              inputCol="vector", outputCol="transformedVector")
+transformer.transform(df).show()
```

Comment:
Could you please update the Scala/Java examples to be the same?  It's nice to call "show()" on the final result.